src/frontend/hooks/useErPerioderSlåttSammen.ts

```diff
@@ -0,0 +1,28 @@
+import { useHttp } from '@navikt/familie-http';
+import { type Ressurs, RessursStatus } from '@navikt/familie-typer';
+import { useCallback, useState } from 'react';
+
+export const useErPerioderSlåttSammen = (behandlingId: string) => {
+    const { request } = useHttp();
+    const [feilmelding, settFeilmelding] = useState<string>();
+    const [erPerioderSlåttSammen, settErPerioderSlåttSammen] = useState<boolean>(false);
+
+    const hentErPerioderSlåttSammen = useCallback(async () => {
+        const response: Ressurs<boolean> = await request<void, boolean>({
+            method: 'GET',
+            url: `/familie-tilbake/api/perioder/erPerioderSlaattSammen/${behandlingId}`,
```

Comment:
```suggestion
            url: `/familie-tilbake/api/perioder/er-sammenslatt/${behandlingId}`,
```

jeg synes dette blir lettere å lese

Reply:
Generelt lurt å unngå case-sensitivitet i url-er ettersom dette går gjennom en del proxy-virksomhet

Reply:
Det var en glipp. Dette endepunktet er slettet som følge av at det ikke var gunstig med to hooks som var avhengige av hverandre.